config/application.rb

```diff
@@ -47,5 +47,7 @@ class Application < Rails::Application
     end
 
     config.middleware.use 'OliveBranch::Middleware'
+
+    config.filter_parameters << :form
```

Comment:
In a worst-case, we could just filter `:education_benefits_claim` which sits one level above `:form`